### PR TITLE
TE-8977: Migrate b2b and b2c public demoshops to Github actions CI

### PR DESCRIPTION
**Pull request by dmytro-dymarchuk**
Ticket: https://spryker.atlassian.net/browse/TE-8977

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  APPLICATION_ENV: ci.pgsql
+  SPRYKER_TESTING_ENABLED: 1
+  APPLICATION_STORE: DE
+  PROJECT: suite
+  POSTGRES_PORT: 5433
+  POSTGRES_PASSWORD: postgres
```
Comment (gechetspr) [inline .github/workflows/ci.yml:18]: Why do we keep env variables here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+  MYSQL_USER: root
+  MYSQL_DATABASE: DE_test_zed
+
+jobs:
+  validation-7-4:
+    name: 'Static analysis / PHP 7.4'
+    runs-on: ubuntu-18.04
+    services:
+      postgres:
+        image: postgres:12
```
Comment (gechetspr) [inline .github/workflows/ci.yml:31]: we use mariadb as default DB. In this case I guess default checks should use it too

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+        run: ./config/Shared/ci/ga/validate.sh
+
+  acceptance-7-4:
+    name: 'Acceptance tests / PHP 7.4'
+    runs-on: ubuntu-18.04
+    services:
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 5672:5672
```
Comment (gechetspr) [inline .github/workflows/ci.yml:72]: if ports are the same, I guess you don't need to map them

File: config/Shared/config_default-ci.mysql.php [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 $config[PropelConstants::ZED_DB_ENGINE] = PropelConfig::DB_ENGINE_MYSQL;
 $config[PropelConstants::ZED_DB_USERNAME] = 'root';
-$config[PropelConstants::ZED_DB_PASSWORD] = '';
+$config[PropelConstants::ZED_DB_PASSWORD] = 'secret';
```
Comment (gechetspr) [inline config/Shared/config_default-ci.mysql.php:12]: do we really need this here? As I can see for public suite we haven't specified passwords in this way

File: tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php [hunk 5]
```diff
@@ -101,8 +101,8 @@ protected function setUp(): void
      */
     protected function skipIfCi()
```
Comment (gechetspr) [inline tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php:102]: Tests works fine with GA?

**Review by gechetspr — CHANGES_REQUESTED** [review]
My main concern is that we don't need to keep passwords in files for public suite, so I'm not sure why do we need them here in this case?

File: tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php (hunk 5 above)
Comment (dmytro-dymarchuk) [reply tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php:102]: Workflow is green

File: config/Shared/config_default-ci.mysql.php (hunk 4 above)
Comment (dmytro-dymarchuk) [reply config/Shared/config_default-ci.mysql.php:12]: It is needed to properly configure DB in GA.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dmytro-dymarchuk) [reply .github/workflows/ci.yml:18]: Variables are used below for configuration db.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (dmytro-dymarchuk) [reply .github/workflows/ci.yml:72]: fixed

File: config/Shared/config_default-ci.mysql.php (hunk 4 above)
Comment (dmytro-dymarchuk) [reply config/Shared/config_default-ci.mysql.php:12]: As I understood it is used only in ci.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (dmytro-dymarchuk) [reply .github/workflows/ci.yml:31]: Renamed steps. Now it looks like checks for MariaDB are default.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (dmytro-dymarchuk) [reply .github/workflows/ci.yml]: But it breaks workflow. According https://docs.github.com/en/actions/guides/about-service-containers#mapping-docker-host-and-service-container-ports

File: tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php (hunk 5 above)
Comment (stereomon) [reply tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php:102]: We don't use CirleCI anymore, we can remove it.

**Review by stereomon — APPROVED** [review]
For me this looks good now., only the CircleCI check left. I personally don't like the sh scripts but moving those to something else will be too much for this ticket.

**Review by gechetspr — APPROVED** [review]

**Pull request merged**